### ISSUE TITLE
Make possible to plugin proxies

### ISSUE THREAD

**Issue by ilyachch** [ilyachch/md_docs-trans-app#44]
Thoughts:
make optional file in config folder like `proxies_list.txt` with list of proxies. If it exists, use them (firstly, try and if it works - use)
make optional launch param to pass list of proxies ips, separated by comma.
make optional launch param to pass file with proxies list (make also possible to set remote file)

**Comment by Prepanpan** [comment]
Yeah.....That's definitely something necessary for Chinese.

**Comment by ilyachch** [comment]
> Yeah.....That's definitely something necessary for Chinese.

@Prepanpan Can you provide a couple of proxy IPs? I need some data I can run tests on

**Comment by Prepanpan** [comment]
> > Yeah.....That's definitely something necessary for Chinese.
> 
> @Prepanpan Can you provide a couple of proxy IPs? I need some data I can run tests on

I am afraid that I cannot :(
The proxy mainland Chinese used cannot be directly expressed by IPs, since it would definitely be blocked by GFW. Mainland Chinese use encrypted algorithms to connect to servers outside mainland China, convert it into proxy running in local PC, and use it to access Google, twitter, etc. For example, the proxy url probably looks like "http://127.0.0.1:1080" or something similar.
If not, Google translate is BLOCKED in mainland China. DeepL is not stable in some areas. Not sure about Bing and Yandex, but Yandex produce nonsense, and Bing is not good either.

**Comment by Prepanpan** [comment]
> > > Yeah.....That's definitely something necessary for Chinese.
> > 
> > 
> > @Prepanpan Can you provide a couple of proxy IPs? I need some data I can run tests on
> 
> I am afraid that I cannot :( The proxy mainland Chinese used cannot be directly expressed by IPs, since it would definitely be blocked by GFW. Mainland Chinese use encrypted algorithms to connect to servers outside mainland China, convert it into proxy running in local PC, and use it to access Google, twitter, etc. For example, the proxy url probably looks like "http://127.0.0.1:1080" or something similar. If not, Google translate is BLOCKED in mainland China. DeepL is not stable in some areas. Not sure about Bing and Yandex, but Yandex produce nonsense, and Bing is not good either.

p.s. Just found out no addition setting is required once I managed to make Google accessible to my computer(automated chrome use the proxy without manually setting).

**Comment by ilyachch** [comment]
> > > > Yeah.....That's definitely something necessary for Chinese.
> > > 
> > > 
> > > @Prepanpan Can you provide a couple of proxy IPs? I need some data I can run tests on
> > 
> > 
> > I am afraid that I cannot :( The proxy mainland Chinese used cannot be directly expressed by IPs, since it would definitely be blocked by GFW. Mainland Chinese use encrypted algorithms to connect to servers outside mainland China, convert it into proxy running in local PC, and use it to access Google, twitter, etc. For example, the proxy url probably looks like "http://127.0.0.1:1080" or something similar. If not, Google translate is BLOCKED in mainland China. DeepL is not stable in some areas. Not sure about Bing and Yandex, but Yandex produce nonsense, and Bing is not good either.
> 
> p.s. Just found out no addition setting is required once I managed to make Google accessible to my computer(automated chrome use the proxy without manually setting).

yeah. but for selenium it is possible to set a separate proxy, that is different from your system. and I was keeping it in mind making this issue
